docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,454 @@
+
+Note that the documentation in this section is intended to show how widgets and state are connected. For a more detailed look see the [Widgets]() reference guide.
+
+### StoreProvider
+
+A `StoreProvider` is a widget that has its own `renderer` and connects to the store. It is always encapsulated in another widget because a it does not have properties defined.
+
+> widget/User.ts
+
+```tsx
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import { WidgetBase } from '@dojo/framework/widget-core/WidgetBase';
+
+import * as css from './styles/Hello.m.css';
+
+interface UserProperties {}
+
+export class User extends WidgetBase<UserProperties> {
+	protected render() {
+		return w<StoreProvider<State>>(StoreProvider, {
+			stateKey: 'state',
+			paths: (path) => [path('users', 'current')],
+			renderer: (store) => {
+				const { get, path } = store;
+				const name = get(path('users', 'current', 'name'));
+
+				return <h1 classes={[css.root]}>{`Hello, ${name}`}</h1>;
+			}
+		});
+	}
+}
+```
+
+The `StoreProvider` occurs as part of `User`'s render and provides its own renderer just like any other widget.
+
+### Container
+
+A `Container` is a widget that fully encapsulates another widget. It connects the store to the widget with a `getProperties` function.
+
+> widget/User.tsx
+
+```tsx
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import { WidgetBase } from '@dojo/framework/widget-core/WidgetBase';
+
+import * as css from './styles/Hello.m.css';
+
+interface UserProperties {
+	name?: string;
+}
+
+export class User extends WidgetBase<UserProperties> {
+	protected render() {
+		const { name = 'Stranger' } = this.properties;
+		return <h1 classes={[css.root]}>{`Hello, ${name}`}</h1>;
+	}
+}
+```
+
+> widget/User.container.ts
+
+```ts
+import { createStoreContainer } from '@dojo/framework/stores/StoreInjector';
+import { Container } from '@dojo/framework/widget-core/Container';
+import { State } from '../interfaces';
+import User from './User';
+
+const StoreContainer = createStoreContainer<State>();
+
+const UserContainer = StoreContainer(User, 'state', {
+	getProperties({ get, path }) {
+		const name = get(path('user', 'current', 'name'));
+
+		return { name };
+	}
+});
+```
+
+In this example `UserContainer` wraps `User` to display the current user's name. `createStoreContainer` is wrapper that is used to ensure the proper typing of `getProperties`. `getProperties` is responsible for getting to data from the store and creating properties for the widget.
+
+The most important thing to note about a `Container` is its properties are a 1:1 mapping to the widget it wraps. In other words, all properties that appear on the widget should be provided by the `Container`.
+
+### Excuting a Process
+
+A process simply defines an execution flow for a set of data. To execute a process it needs access to the store to create an executor. Both the `Container` and `StoreProvider` widgets will provide you with access to the store.
```

Comment:
"To execute a process it needs access to the store to create an executor." replace it with the term (presumably process?)